pkg/git.go

```diff
@@ -48,6 +48,8 @@ func NewGitPackage(source *deps.Git) Interface {
 	}
 }
 
+var GitQuiet = false
```

Comment:
I don't particularly like globals, but refactoring this code is really out of scope here and I can't think of a better solution given the current way things are